app/_how-tos/kic-sticky-session-with-drain-support.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Configuring Sticky Sessions with Drain Support
+description: |
+  Learn how to implement sticky sessions with graceful draining of terminating pods using {{site.kic_product_name}}.
+  
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Guides
+
+permalink: /kubernetes-ingress-controller/sticky-sessions-with-drain-support/
+
+content_type: how_to
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+tldr:
+  q: How do I enable sticky sessions with drain support in {{ site.kic_product_name }}?  
+  a: Deploy {{ site.kic_product_name }} using the `--enable-drain-support=true` flag. Next, configure `spec.stickySessions` and set `spec.algorithm` to `sticky-sessions` in a `KongUpstreamPolicy` resource. Finally, attach the `KongUpstreamPolicy` resource to a Kubernetes Service with the `konghq.com/upstream-policy` annotation.
+
+prereqs:
+  kubernetes:
+    gateway_api: true
+  entities:
+    services:
+      - httpbin-service
+    routes:
+      - httpbin
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+
```

Comment:
Can we try adding 


```suggestion

min_version:
  kic: '3.5'
```

as suggested by Michael in this thread: https://kongstrong.slack.com/archives/CDSTDSG9J/p1751895500399979?

This will ensure that this guide is only for KIC 3.5+.